File: app/src/main/java/com/example/learnit/word.java
package com.example.learnit;

public class word {

    private String defaultTranslation;
    private String frenchTranslation;
    private int imageResourceID;

    public word(String defaultTranslation, String frenchTranslation) {
        this.defaultTranslation = defaultTranslation;
        this.frenchTranslation = frenchTranslation;
        imageResourceID = 0;
    }

    public word(String defaultTranslation, String frenchTranslation, int imageResourceID) {
        this.defaultTranslation = defaultTranslation;
        this.frenchTranslation = frenchTranslation;
        this.imageResourceID = imageResourceID;
    }

    public String getDefaultTranslation() {
        return defaultTranslation;
    }

    public String getFrenchTranslation() {
        return frenchTranslation;
    }

    public int getImageResourceID() {
        return imageResourceID;
    }
}

File: app/src/main/java/com/example/learnit/word.kt
package com.example.learnit

class word {
    var defaultTranslation: String
        private set
    var frenchTranslation: String
        private set
    var imageResourceID: Int
        private set
    var audioResourceId: Int
        private set

    constructor(defaultTranslation: String, frenchTranslation: String) {
        this.defaultTranslation = defaultTranslation
        this.frenchTranslation = frenchTranslation
        imageResourceID = 0
        audioResourceId = 0
    }

    constructor(defaultTranslation: String, frenchTranslation: String, imageResourceID: Int, audioResourceId: Int) {
        this.defaultTranslation = defaultTranslation
        this.frenchTranslation = frenchTranslation
        this.imageResourceID = imageResourceID
        this.audioResourceId = audioResourceId
    }
}